.env.example

```diff
@@ -0,0 +1,2 @@
+#REACT_APP_URL=http://neighborhoodchef-be-prod.us-east-1.elasticbeanstalk.com/graphql
```

Comment:
this is commented out for now, url will change after redeploy, will need to change this accordingly, then uncomment it, and comment line 2 instead for final draft of .env.example